<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <arg name="sub_topic_name" default="nmea_sentence"/>
  <arg name="pub_fix_topic_name" default="fix"/>
  <arg name="pub_gga_topic_name" default="gga"/>
  <arg name="pub_rmc_topic_name" default="rmc"/>
  <arg name="output_gga" default="true"/>
  <arg name="output_rmc" default="false"/>

  <node pkg="eagleye_nmea2fix" name="nmea2fix_node" type="nmea2fix_node" output="screen">
    <param name="sub_topic_name" value="$(arg sub_topic_name)"/>
    <param name="pub_fix_topic_name" value="$(arg pub_fix_topic_name)"/>
    <param name="pub_gga_topic_name" value="$(arg pub_gga_topic_name)"/>
    <param name="pub_rmc_topic_name" value="$(arg pub_rmc_topic_name)"/>
    <param name="output_gga" value="$(arg output_gga)"/>
    <param name="output_rmc" value="$(arg output_rmc)"/>
  </node>

</launch>
